gallery/src/styles/NxStatusIndicator/NxStatusIndicatorPage.tsx

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxCode, NxP, NxReadOnly, NxTile, NxH3 } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxStatusIndicatorComponentExample from './NxStatusIndicatorComponentExample';
+
+const nxStatusIndicatorExampleCode = require('./NxStatusIndicatorExample.html'),
+    nxStatusIndicatorComponentExampleCode = require('./NxStatusIndicatorComponentExample?raw');
+
+const NxThreatIndicatorPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>nx-status-indicator</NxCode> is used to indicate four distinct states of a workflow or external system.
+        Depending on the usage scenario, the exact wording associated with each state may vary.
+        The associated wording must be specified as the text content of the element to which
+        the <NxCode>nx-status-indicator</NxCode> class is applied.
+      </NxP>
+      <NxP>
+        Unlike tags, <NxCode>nx-status-indicator</NxCode>s do not appear conditionally. They are always visible and
+        indicate one of the
+        four distinct states to reflect related object's current state. They should not be used to
+        report error occurrences.
+      </NxP>
+      <NxReadOnly>
+        <NxReadOnly.Label>Example</NxReadOnly.Label>
+        <NxReadOnly.Data>
+          Scenario – A REST call has failed (an event)<br/>
+          Do not use nx-status-indicator
+        </NxReadOnly.Data>
+        <NxReadOnly.Data>
+          Scenario – A REST service is broken and unresponsive (a state)<br/>
+          Use <NxCode>nx-status-indicator</NxCode> with the <NxCode>error</NxCode> state
+        </NxReadOnly.Data>
+      </NxReadOnly>
+      <NxP>
+        The table below describes the four states that <NxCode>nx-status-indicator</NxCode> can represent as well as
+        suggested paired wording for each state:
+      </NxP>
+      <NxTable>
+        <NxTable.Head>
+          <NxTable.Row>
+            <NxTable.Cell>State</NxTable.Cell>
+            <NxTable.Cell>Description</NxTable.Cell>
+            <NxTable.Cell>Suggested Paired Wording</NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Head>
+        <NxTable.Body>
+          <NxTable.Row>
+            <NxTable.Cell><NxCode>negative</NxCode></NxTable.Cell>
+            <NxTable.Cell>The item in question is absent or not activated. </NxTable.Cell>
+            <NxTable.Cell>
+              "Off", "Disabled", "Inactive", "Not Started", "Down"
```

Comment:
How about adding "Offline"? The status indicator could be useful for indicating whether a server is "Online" or "Offline".